tools/alpine/push-manifest.sh

```diff
@@ -46,7 +46,12 @@ EOF
 # we need them for notary on all platforms.
 case $(uname -s) in
     Darwin)
-        CRED=$(echo "https://index.docker.io/v1/" | /Applications/Docker.app/Contents/Resources/bin/docker-credential-osxkeychain.bin get)
+        # DfM recently (2018-03-27) change the mac key chain utility binary name. Handle both for now
+        if [ -f /Applications/Docker.app/Contents/Resources/bin/docker-credential-osxkeychain.bin ]; then
+            CRED=$(echo "https://index.docker.io/v1/" | /Applications/Docker.app/Contents/Resources/bin/docker-credential-osxkeychain.bin get)
+        else
+            CRED=$(echo "https://index.docker.io/v1/" | /Applications/Docker.app/Contents/Resources/bin/docker-credential-osxkeychain get)
```

Comment:
Same change is needed in `script/push-manifest.sh`, I think. After that you'd want to run `go generate github.com/linuxkit/linuxkit/src/cmd/linuxkit/pkglib` too.

Your comment suggests you think this change might be reverted? If that is likely then fine, if it is unlikely then a comment like `# Prior to 2018-03-27 D4M used a .bin suffix on the keychain utility binary name` would be more appropriate.